Pod/Classes/VOKCoreDataManager.m

```diff
@@ -161,8 +145,7 @@ - (void)initPersistentStoreCoordinator
                               NSInferMappingModelAutomaticallyOption: @YES,
                               };
     
-    NSError *error;
-    _persistentStoreCoordinator = [[NSPersistentStoreCoordinator alloc] initWithManagedObjectModel:[self managedObjectModel]];
+    _persistentStoreCoordinator = [[NSPersistentStoreCoordinator alloc] initWithManagedObjectModel:self.managedObjectModel];
```

Comment:
I worry that the getter methods would become bloated and potentially confusing.  How about making the init methods into factory/create methods similar to what I did with `managedObjectContextWithConcurrencyType:`?